ns1/permissions.go

```diff
@@ -56,6 +31,8 @@ func addPermsSchema(s map[string]*schema.Schema) map[string]*schema.Schema {
 			},
 		},
 	}
+	s["dns_records_allow"] = dnsRecords
+	s["dns_records_deny"] = dnsRecords
```

Comment:
Shouldn't each of these be separate pointers to a schema struct?  As is I think they are both referencing the same pointer, which doesn't seem like what we want.

Reply:
@mburtless Since this is a schema that tells terraform how to parse the data from the .tf file, from my, understanding we can use the same pointer for both since they will be parsed in the same way, this doesn't affect the functionality, I ran the tests and also did some manual tests and works fine.